src/EventExport/Format/TabularData/TabularDataEventFormatter.php

```diff
@@ -767,12 +768,9 @@ private function parseEventIdFromUrl($event): string
      * Gives a formatter that tries to fetch a summary in plain text.
      * If the formatted summary is missing, the summary that is available on the event will be used as fallback.
      *
-     * @param Format                                  $format
-     * @param CalendarSummaryRepositoryInterface|null $calendarSummaryRepository
-     *
-     * @return string
+     * @return Closure
```

Comment:
I think the `Closure` class is usable in 7.1 so maybe we should just add that on the method itself like you did for the `CalendarSummaryRepositoryInterface` argument? 

https://www.php.net/manual/en/class.closure.php
> (PHP 5 >= 5.3.0, PHP 7, PHP 8)